packages/react/src/components/button.tsx

```diff
@@ -31,60 +32,58 @@ const button_cva = cva(
         "default-outline": [
           "bg-bg-white border border-otl-gray-200 text-txt-black-700 shadow-button",
           "hover:bg-bg-white-hover hover:border-otl-gray-300 hover:text-txt-black-900",
-          "focus:ring ring-fr-primary/0 focus:ring-fr-primary",
+          "focus:ring focus:ring-fr-primary",
           "disabled:bg-bg-white-disabled disabled:text-txt-black-disabled disabled:border-transparent",
         ],
         "default-ghost": [
           "bg-transparent border border-transparent text-txt-black-700",
           "hover:bg-bg-white-hover",
-          "focus:ring ring-fr-primary/0 focus:ring-fr-primary",
+          "focus:ring focus:ring-fr-primary",
           "disabled:bg-bg-white-disabled disabled:text-txt-black-disabled disabled:border-transparent",
         ],
         "danger-fill": [
           "bg-danger-600 border border-danger-600 text-white shadow-button",
           "hover:bg-danger-700",
-          "focus:ring ring-fr-primary/0 focus:ring-fr-primary",
+          "focus:ring focus:ring-fr-danger",
           "disabled:bg-bg-danger-disabled disabled:text-white-disabled disabled:border-bg-danger-disabled",
         ],
         "danger-outline": [
           "bg-bg-white border border-otl-danger-200 text-txt-danger shadow-button",
           "hover:bg-bg-danger-50",
-          "focus:ring ring-fr-primary/0 focus:ring-fr-primary",
+          "focus:ring focus:ring-fr-danger",
           "disabled:bg-bg-white-disabled disabled:text-txt-danger-disabled disabled:border-transparent",
         ],
         "danger-ghost": [
           "bg-transparent border border-transparent text-txt-danger",
           "hover:bg-bg-danger-50",
-          "focus:ring ring-fr-primary/0 focus:ring-fr-primary",
+          "focus:ring focus:ring-fr-danger",
           "disabled:bg-bg-white-disabled disabled:text-txt-danger-disabled disabled:border-transparent",
```

Comment:
Why change to `ring-fr-danger`? The `fr-primary` is following the Figma spec: https://www.figma.com/design/svmWSPZarzWrJ116CQ8zpV/MYDS-(Beta)?node-id=11-11293&node-type=canvas&t=oijp5bsFSusOoHwK-0

Reply:
ah my bad, was testing cause i thought it was kinda odd, will change it back to primary